### PR TITLE
[deckhouse-controller] Changelogs for modules

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Description
Added changelog to deckhouse release

## Why do we need it, and what problem does it solve?
https://github.com/deckhouse/deckhouse/issues/6847

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```ch.anges
section: deckhouse-controller
type: feature
summary: Added changelog to deckhouse release
impact_level: default
```

### CODE REVIEW

**Comment by libmonsoon-dev** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/static'...

File: deckhouse-controller/crds/doc-ru-module-release.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ spec:
                   description: Время, до которого отложено обновление.
                 requirements:
                   description: Структура содержащая требования, необходимые для применения релиза.
+                changelog:
+                  description: Изменения модуля в данном релизе.
```
Comment (z9r5) [inline deckhouse-controller/crds/doc-ru-module-release.yaml:24]:
```suggestion
                  description: Список изменений модуля в данном релизе.
```
@libmonsoon-dev Это список будет список на основе PR, относящихся к модулю, да?

Comment (libmonsoon-dev) [reply deckhouse-controller/crds/doc-ru-module-release.yaml]: Да, все верно

**Review by z9r5 — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Pull request merged**